### PR TITLE
Project page text reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: project.html [hunk 1]
```diff
@@ -4,38 +4,38 @@
 permalink: project
 ---
 
-<p><strong>KeePassXC</strong> is a modern, secure, and open-source password manager that stores and manages your most sensitive information.</p>
+<p><strong>KeePassXC</strong> is a modern, secure, and open-source password manager that stores and manages your most sensitive info.</p>
```
Comment (phoerious) [inline project.html:7]: Why is info better here than information? It's the same, just more informal and less precise.

File: project.html [hunk 2]
```diff
@@ -4,38 +4,38 @@
...
 
-<p>You can run KeePassXC on Windows, macOS, and Linux systems. KeePassXC is for people with extremely high demands of secure
-personal data management. It saves many different types of information, such as usernames, passwords, URLs, attachments,
-and notes in an offline, encrypted file that can be stored in any location, including private and public cloud solutions.</p>
+<p>For Linux, macOS and Windows. Save many different types of info (such as usernames, passwords, URLs, attachments,
```
Comment (phoerious) [inline project.html:9]: Again, I really don't like info. Why not information or data?

File: project.html [hunk 3]
```diff
@@ -4,38 +4,38 @@
...
+and notes) in an offline, encrypted file that can be stored on your device, computer, server, or the cloud.</p>
 
-<p>For easy identification and management, user-defined titles and icons can be specified for entries. In addition, entries
-are sorted in customizable groups. An integrated search function allows you to use advanced patterns to easily find any entry
-in your database. A customizable, fast, and easy-to-use password generator utility allows you to create passwords with any
-combination of characters or easy to remember passphrases.</p>
+<p>Identify and manage entries with user-defined titles and icons, and customizable groups.
+Use advanced patterns in the integrated search function to easily find any entry in your database.
```
Comment (phoerious) [inline project.html:13]: Advanced queries.

File: project.html [hunk 4]
```diff
@@ -4,38 +4,38 @@
...
+Generate passwords with any combination of characters or easy to remember passwords by using the password generator.</p>
```
Comment (phoerious) [inline project.html:14]: easy-to-remember

File: project.html [hunk 5]
```diff
@@ -4,38 +4,38 @@
...
 
 <h3>Quick Start</h3>
-<p>The <a href="/docs/KeePassXC_GettingStarted.html">QuickStart Guide</a> gets you started using KeePassXC on your
-Windows, macOS, or Linux computer using pre-compiled binaries from the <a href="/download">downloads page</a>.
-Additionally, individual Linux distributions may ship their own versions, so please check your distribution's package
-list to see if KeePassXC is available. Detailed documentation is available in the <a href="/docs/KeePassXC_UserGuide.html">User Guide</a>.</p>
+<p>The <a href="/docs/KeePassXC_GettingStarted.html">quickStart guide</a> explains using KeePassXC on your
```
Comment (phoerious) [inline project.html:17]: QuickStart Guide explains how to

File: project.html [hunk 6]
```diff
@@ -4,38 +4,38 @@
...
+Linux, macOS, or Windows computer by <a href="/download">downloading</a> and then installing it.
+Various other Linux distributions ship their own versions, so check the package
+list of yours to see if KeePassXC is available.
+The <a href="/docs/KeePassXC_UserGuide.html">user guide</a> has detailed documentation.</p>
```
Comment (phoerious) [inline project.html:21]: User Guide

File: project.html [hunk 7]
```diff
@@ -4,38 +4,38 @@
...
 
 <h3>Features List</h3>
-<p>KeePassXC has numerous features for novice and power users alike. Our goal is to create an application that can be used
-by anyone while still offering advanced features to those that need them.</p>
+<p>Numerous features for novice and power users alike to make a solution everyone
+can use, while still offering advanced features to those that need them.
+It even works for people with extremely high demands of secure personal data management.</p>
```
Comment (phoerious) [inline project.html:26]: Sounds weird. I would rephrase: "It is a solution for your everyday use as well as for people with extremely high demands on secure personal data management."

File: project.html [hunk 8]
```diff
@@ -4,38 +4,38 @@
...
 
 <div id="features">
     <h4>Basic</h4>
     <ul>
         <li>Create, open, and save databases in the KDBX format (KeePass compatible to KDBX4 and KDBX3)</li>
-        <li>Store sensitive information in entries that are organized by groups</li>
+        <li>Store sensitive info in entries organized in groups</li>
```
Comment (phoerious) [inline project.html:32]: data

**Review by phoerious — CHANGES_REQUESTED** [review]

File: project.html (hunk 1 above)
Comment (comradekingu) [reply project.html:7]: It is shorter. Just as precise.

File: project.html (hunk 2 above)
Comment (comradekingu) [reply project.html:9]: Data is less tangible. I think that is a judgement call.

File: project.html (hunk 3 above)
Comment (comradekingu) [reply project.html:13]: Maybe, but I think it complicates matters. It is less often that people know the word "query"(?)

File: project.html (hunk 6 above)
Comment (comradekingu) [reply project.html:21]: I think that is optional, and an error that leaks into translations where it isn't.

File: project.html (hunk 8 above)
Comment (comradekingu) [reply project.html:32]: It is the same as above, either has to be data in both places, or info.

File: project.html (hunk 7 above)
Comment (comradekingu) [reply project.html:26]: Too long.

File: project.html (hunk 4 above)
Comment (comradekingu) [inline project.html:14]:
```suggestion
Generate passwords with any combination of characters or easy-to-remember passwords by using the password generator.</p>
```

File: project.html (hunk 5 above)
Comment (comradekingu) [inline project.html:17]:
```suggestion
<p>The <a href="/docs/KeePassXC_GettingStarted.html">quickstart guide</a> explains using KeePassXC on your
```

File: project.html (hunk 7 above)
Comment (comradekingu) [inline project.html:26]:
```suggestion
It even suits people with extremely high secure data management demands.</p>
```

File: project.html (hunk 3 above)
Comment (comradekingu) [inline project.html:13]:
```suggestion
Use the advanced search function to easily find any entry in your database.
```
